src/stories/Library/material-search/material-search.scss

```diff
@@ -0,0 +1,235 @@
+// Variables
+$_gin_xs_spacing: 0.5rem;
+$_gin_m_spacing: 1rem;
+$_gin_border_color: #8e929c;
+$_gin_font_weight_semibold: 525;
+
+$_color__background-light: #eeeeed;
+$_color__hover-background: #8e929c;
+
+$_box-shadow1: 1px 1px 5px 1px rgba(0, 0, 0, 0.1);
+$_box-shadow: 0px 0px 0px 2px rgba(0, 0, 0, 0.25);
+$_loading-spinner-size: 40px;
+
+// These heights are explicitly set to covers used in the material search.
+// The reason for this is, that the covers must all use "large" as their size
+// prop, because this prop is used to determine which size of image is
+// displayed in the cover as provided by cover-service.
+// Example: A small "size" prop is used on the listResults and preview in
+// material search, but the component being used for displaying the materials
+// (I.e RecommendedMaterial) has its' prop set to "large", then image will
+// potentially not be displayed at all, since a "large" version is not served
+// by cover-service, but a small version is.
+
+// Arguably, this is a flaw in cover service, and should be fixed there, but
+// since there is no solution to this, and it potentially could happen for the
+// vast majority of materials I have decided to manually overwrite this here,
+// to ensure that the covers are displayed correctly and avoid frustration.
+$MATERIAL_SMALL_MOBILE: 104px;
+$MATERIAL_SMALL_DESKTOP: 137px;
+
+$list_header_height: 50px;
+
+// Material Search
+.material-search {
+  max-width: 800px;
+  margin-bottom: $_gin_m_spacing;
+}
+
+.material-search__label {
+  display: flex;
+  flex-direction: column;
+  gap: $_gin_xs_spacing;
+  margin-bottom: $_gin_xs_spacing;
+  font-weight: $_gin_font_weight_semibold;
+}
+
+.material-search__inputs-container {
+  display: grid;
+  grid-template-columns: 2fr minmax(auto, 220px);
+  gap: $_gin_m_spacing;
+  margin-bottom: $_gin_xs_spacing;
+}
+
+.material-search__inputs-container--hidden {
+  display: none; // This class is only applied in Drupal.
+}
+
+// Preview
+.material-search__preview {
+  color: $color__global-grey;
+  cursor: not-allowed;
+  display: grid;
+  background-color: $color__global-white;
+  min-height: 170px;
+  align-items: center;
+  border: 1px solid $_gin_border_color;
+  border-radius: $_gin_xs_spacing;
+  overflow: hidden;
+}
+
+.material-search__preview-material {
+  display: grid;
+  padding: 0 $_gin_m_spacing;
+  grid-template-columns: min-content auto;
+  column-gap: $_gin_m_spacing;
+  align-items: center;
+  height: 100%;
+
+  .cover {
```

Comment:
Normally this nesting would not fly.

I recognize the need for it based on the comment in the top. I wonder if it should be fixed in the cover components through a modifier there instead of this nesting 🤔 .

Reply:
I have refactored cover to take a className being added to the cover wrapper. This way we can pass a specific BEM class to the component. I think this is simpler.